sys/src/9/ufs/ufs/ufs_inode.c

```diff
@@ -47,7 +54,19 @@
 #endif
 #ifdef UFS_GJOURNAL
 #include <ufs/ufs/gjournal.h>
-#endif
+#endif*/
+
+
+/*
+ * Conversion tables for conversion from vnode types to inode formats
+ * and back.
+ */
+static Vtype iftovt_tab[16] = {
```

Comment:
wow, they have all these types? The only ones in there should be DIR, REG and, maybe, LNK